lib/active_merchant/billing/gateways/payeezy.rb

```diff
@@ -247,13 +247,13 @@ def add_network_tokenization(params, payment_method, options)
         nt_card[:card_number] = payment_method.number
         nt_card[:exp_date] = format_exp_date(payment_method.month, payment_method.year)
         nt_card[:cvv] = payment_method.verification_value
-        nt_card[:xid] = payment_method.payment_cryptogram
-        nt_card[:cavv] = payment_method.payment_cryptogram
+        nt_card[:xid] = payment_method.payment_cryptogram unless payment_method.payment_cryptogram.empty?
+        nt_card[:cavv] = payment_method.payment_cryptogram unless payment_method.payment_cryptogram.empty?
```

Comment:
Do these need to be set to `nil` or omitted from being sent if there is no `payment_cryptogram`?

Reply:
Does this impact/benefit all customers of this gateway type (i.e.: is it an issue every customer faces presently) or just the one  customer (Priceline?)

Reply:
We were explicitly told by Payeezy to NOT send `nil` values for these fields as they "could" lead to errors. These fields are `optional` according to Payeezy's documentation.

AFAIK, Priceline is the only user of AP on Payeezy, but this should not negatively impact any other customers of this gateway.

Reply:
@chudel this would benefit all customers of this gateway type